### PR TITLE
Feature/101/change kids profile

### PR DESCRIPTION
**Pull request by Alaleh-kash**
# PR Template Frontend

## One Line Description
Create Dropdown menu for changing kids profile (Upload Photo, Change Color, Remove Photo)

## Requirements
- Clearly describe the requirements and functionalities expected from the code changes.
- Requirements should be derived from the issue description.

## Notes
Any additional information relevant to the PR, such as context, reasons for changes, or anything you want to draw attention to, like the scope of the ticket.

## Test Steps
- Step-by-step instructions on how to test the new features or fixes.
- Include any specific commands or setups needed.

## UI/UX Outcome
- Add any relevant screenshots or videos to demonstrate the UI/UX changes.

## Checklist
- [x] Pull Request title includes the issue number and a brief description of the change.
- [x] All changes should be tested and verified to work correctly.
- [x] Code follows frontend best practices.
- [x] Branch names follow the conventions in the contributing file.
- [x] Commit messages follow the naming conventions in the contributing file.

### CODE REVIEW

File: src/components/shared/ui/Button/CustomButton.jsx
Comment (swathivt) [inline src/components/shared/ui/Button/CustomButton.jsx:1]: You've done good work, but some changes are needed. Don't change the customButton.jsx file. If you want to modify the styles of the customButton, make changes in the file where you are importing it, such as messageModal.jsx.

File: src/components/shared/ui/Button/buttonSizes.css
Comment (swathivt) [inline src/components/shared/ui/Button/buttonSizes.css:1]: Do not modify the buttonSizes style file either

File: src/components/shared/ui/Button/contentStyles.css
Comment (swathivt) [inline src/components/shared/ui/Button/contentStyles.css:1]: Do not modify the buttonSizes style file either

**Comment by Alaleh-kash** [comment]
Hi @swathivt, just to clarify, I didn't modify the customButton.jsx, buttonSizes.css, or contentStyles.css files. I only imported them into KidForm.jsx. Should I proceed with styling modifications directly in KidForm.jsx?

File: src/components/shared/ui/Button/CustomButton.jsx
Comment (Alaleh-kash) [reply src/components/shared/ui/Button/CustomButton.jsx:1]: Hi @swathivt, Thank you for your feedback on my work. I make necessary adjustment.

**Pull request closed without merging**